api/rpc/account/account.proto

```diff
@@ -0,0 +1,248 @@
+syntax = "proto3";
+
+package account;
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service AccountService {
+  rpc SignUp (SignUpRequest) returns (SessionReply) {
+    option (google.api.http) = {
+      post: "/v1/account/{name}/signup",
+      body: "*"
+    };
+  };
+  rpc Verify (VerificationRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      post: "/v1/account/{name}/verification/{code}",
+      body: "*"
+    };
+  };
+  rpc CreateOrganization (OrganizationRequest) returns (google.protobuf.Empty) {
```

Comment:
Agree. The "ID" is the username that was specified. We don't need to surface the internal id.